### PR TITLE
Uses AST-scanner for all ERB

### LINKED ISSUE #572: Version 1.0.14 thinks any method ending in `_t` is a translation method
I have a method `theme_t` that takes a string and is used in my ERB files. With version 1.0.14, i18n-tasks has started complaining that these string arguments cannot be found in the translation files.

It seems the relevant change is e939d0e1f51baaa5d8f229f54bc18d3cc438d71b, since the changed regex matches `theme_t "foo"` but the original regex did not.

### PR DESCRIPTION
**Pull request by davidwessman**
- Also fixes regex in PatternScanner to handle `theme_t "foo"` being
  marked as a translation.
- Fixes #572

### CODE REVIEW

**Comment by davidwessman** [comment]
@mvz Can you test it out? 🙂

File: CHANGES.md [hunk 1]
```diff
@@ -1,83 +1,88 @@
+## Unreleased
+
+- Uses AST-parser for all ERB-files, not just `.html.erb`
+- [Fixed regex in `PatternScanner`] (https://github.com/glebm/i18n-tasks/issues/572)
+
 ## v1.0.14
 
-* Newlines are now preserved when using Google Translate.
+- Newlines are now preserved when using Google Translate.
```
Comment (glebm) [inline CHANGES.md:8]: Can you please revert the changes from `-` to `*`?

**Comment by glebm** [comment]
Looks good to me other than the markdown formatting change

**Comment by davidwessman** [comment]
> Looks good to me other than the markdown formatting change

Need to find some better settings for the VS Code repo to avoid automatic formatting.

**Comment by mvz** [comment]
@davidwessman I have tested with these changes and it works now. Thanks!

**Comment by gregorbg** [comment]
I'm a little worried that this may cause regressions for https://github.com/glebm/i18n-tasks/issues/537.

The limitation to only `.html.erb` was deliberately introduced here: https://github.com/glebm/i18n-tasks/pull/543
This current PR looks like it exactly reverts the fix from that other PR. What will happen to `.js.erb` files again when this change is released?

**Comment by davidwessman** [comment]
> I'm a little worried that this may cause regressions for https://github.com/glebm/i18n-tasks/issues/537.
> 
> 
> 
> The limitation to only `.html.erb` was deliberately introduced here: https://github.com/glebm/i18n-tasks/pull/543
> 
> This current PR looks like it exactly reverts the fix from that other PR. What will happen to `.js.erb` files again when this change is released?

There is a new scanner that handles it 🙂 and there are tests for it.

**Comment by davidwessman** [comment]
But please add some tests if you think there needs to be more test coverage for it.

**Comment by davidwessman** [comment]
https://github.com/glebm/i18n-tasks/pull/545

Here is the new parser 🙂
